### ISSUE TITLE
Error decrypting file from server

### ISSUE THREAD

**Issue by ahaeslich** [sandstorm/synco#24]
I already tested the package successfully last year. But with the current version I get this error while trying to download the files.

```
 SUCCESS  Found correct base URL at https://my.public..omain/ (based on .synco.yml)
 ERROR   Metadata could not be fetched: error decrypting file from server - most likely, the encryption key was wrong: no identity matched any of the recipients
```

**Comment by skurfuerst** [comment]
@ahaeslich confirmed, I have the same error. Weird, will investigate :)

**Comment by skurfuerst** [comment]
@ahaeslich fixed with 1.1.5 :) Thanks for reporting! Problem was introduced in https://github.com/sandstorm/synco/commit/9a19edc3da6d8f8cef714f29a0636940e9416967
